generators/server/templates/gradle.properties.ejs

```diff
@@ -48,7 +48,7 @@ log4j2_mock_version=0.0.2
 jackson_databind_nullable_version=<%= JACKSON_DATABIND_NULLABLE_VERSION %>
 <%_ } _%>
 <%_ if (cacheProvider === 'caffeine') { _%>
-caffeine_version=2.8.1
+caffeine_version=2.8.4
```

Comment:
Can we store this version in `generators/constants.js` instead of having it here and in the Maven pom independently?  Once it is there it is accessible via the `versions` import above

Reply:
Sure no problem. Maybe we can backport that to the main generator afterwards.